serverFiles/endpoints.json

```diff
@@ -53,5 +53,16 @@
       "votes": 0,
       "created_at": "2020-11-03T21:00:00.000Z"
     }
+  },
+  "POST /api/articles/:article_id/comments": {
```

Comment:
Since this post request requires a body from the user, it would be nice to add a section to show what an example request should look like

Reply:
thanks lewis, i made all those changes